zerver/migrations/0570_directmessagegroup_group_size.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 5.0.6 on 2024-08-16 06:36
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("zerver", "0569_remove_userprofile_tutorial_status"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="directmessagegroup",
+            name="group_size",
+            field=models.IntegerField(db_index=True, null=True),
+        ),
+    ]
```

Comment:
Generally you'll want a third migration that removes `null=True`, created by editing `recipients.py` to remove it and running `manage.py makemigrations` after. You'll see a lot of examples just after migrations with matches to `git grep -i batch_size`.

Reply:
Ya right, removing null=True actually made me realize places where the `group_size` field was actually not being set -- like in mattermost, rocketchat importers.